pkg/workloads/spark_job/spark_util.py

```diff
@@ -48,6 +48,15 @@
     consts.COLUMN_TYPE_STRING_LIST: [ArrayType(StringType(), True)],
 }
 
+CORTEX_TYPE_TO_CASTABLE_SPARK_TYPES = {
+    consts.COLUMN_TYPE_INT: [IntegerType(), LongType()],
+    consts.COLUMN_TYPE_INT_LIST: [ArrayType(IntegerType(), True), ArrayType(LongType(), True)],
+    consts.COLUMN_TYPE_FLOAT: [FloatType(), DoubleType()],
+    consts.COLUMN_TYPE_FLOAT_LIST: [ArrayType(FloatType(), True), ArrayType(DoubleType(), True)],
+    consts.COLUMN_TYPE_STRING: [StringType(), IntegerType(), LongType(), FloatType(), DoubleType()],
```

Comment:
Should this be a catch-all? Can we cast any type to string if the user asked for it?

Reply:
Yes.